OneCommunityGlobal/HighestGoodNetworkApp
# Set Task tab as default after login

# Fixes PROJECTS/TASKS/WBS COMPONENT (BUG #14 PRIORITY LOW)

This PR intends to set the Tasks tab as default when acessing the Dashboard. So the Task tab should automatically open after the login and when returning to Dashboard after navigating the App. This is suposed to happen to all account roles except a volunteer account with **no** task wbs assign to it. Volunteer accounts **with wbs tasks** should have Tasks tab as default too.

## Related PR's
This frontend PR is related to the old [#535](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/535) frontend PR. I took over to completed it but there were too many conflicts to be safely merged so raising a new PR was safer.

## How to test:
Keep backend on development
check into this PR on frontent
run `git pull` and `npm install`
log as different account roles and test if with all of them (except volunteer with no WBS) the Tasks tab is shown first.
Assign a **WBS task** to a volunteer account and test if the Task tab is being displayed first.

![image](https://user-images.githubusercontent.com/65736872/221022404-84ba38d0-85c6-48bc-87a1-2dc4e8183714.png)

## Note:
Please observe that it's necessary to have a **WBS task** assigned to it not just a Project. If you're not sure how to do it, please check video below.


https://user-images.githubusercontent.com/65736872/221026449-4a37f61e-a1e8-4099-9cc8-e963ceb947f7.mp4
